### DISCUSSION TITLE
Weekly Task (1) - Prerequisites

### DISCUSSION THREAD

**Discussion by akash-coded** [akash-coded/mern#2] [Tasks]
1. Go through the basic concepts of Git and GitHub.

2. Create a GitHub account and star the common repo using the [link](https://github.com/akash-coded/mern) shared with you.

3. Install VSCode in your system.

**Comment by Priyavs-coder** [comment]
Hi sir. I'm Shanmugapriya. I have created my own Github account and installed Visual studio code @akash-coded

**Comment by HARSH062001** [comment]
HELLO SIR MY SELF HARSH AND ALREADY HAVE MY GITHUB ACCOUNT AND Visual Studio Code installed @akash-coded

**Comment by Divyam-09** [comment]
@akash-coded 
Good Evening Sir,
Myself Divya
I have created my GitHub login and installed VS Code

**Comment by ghost** [comment]
@akash-coded 
Hello sir,
My name is VIshwajeet Shukla and I have installed VS Code and created my Github account.

**Comment by KNiranjani** [comment]
@akash-coded 
Hi Sir, I'm Niranjani, I've Created My GitHub Account and Installed VS Code Software in my system

**Comment by MarioCodes29** [comment]
Hi @akash-coded Sir,
It's Allwyn Nadar
 I've created my GitHub account and Installed VS code IDE

**Comment by Vasustark** [comment]
Hi @akash-coded sir
I'm Vasudevan.
I've created my github account and installed VS code.

**Comment by Sridhar-Kesari** [comment]
@akash-coded 
Created github account and installed VS code.

**Comment by shubhamk-coder** [comment]
@akash-coded Hello,
I have completed all the tasks.
VSCode was already Installed
The Github account was already there 
And I have installed Git now

**Comment by Tejeshcy** [comment]
Hi @akash-coded sir
I'm Tejesh.
I've created my github account and installed VS code.

**Comment by Neethumar** [comment]
Hi @akash-coded sir
I have created github account and installed VS code

**Comment by jo300** [comment]
Hi @akash-coded sir
I have created github account and installed VS code

**Comment by alfiyariyaz18** [comment]
Hi @akash-coded,
Sir I have created my Github account

**Comment by sienadsouza** [comment]
Hi @akash-coded 
Sir I have created my Github account

**Comment by Aayush217** [comment]
Hi @akash-coded,
I've created my Github account and installed VS code

**Comment by Shalvi187** [comment]
Hi @akash-coded,
I have already created my GitHub account and installed VsCode Editor on my system.

**Comment by Nikhi54** [comment]
I have already created my github account and installed Vscode editor on my system
